proto/buf/alpha/registry/v1alpha1/authz.proto

```diff
@@ -84,7 +94,9 @@ service AuthzService {
   rpc UserCanManagePluginContributors(UserCanManagePluginContributorsRequest) returns (UserCanManagePluginContributorsResponse);
   // UserCanManageTemplateContributors returns whether the user is authorized to manage
   // any contributors to the template and the list of roles they can manage.
-  rpc UserCanManageTemplateContributors(UserCanManageTemplateContributorsRequest) returns (UserCanManageTemplateContributorsResponse);
+  rpc UserCanManageTemplateContributors(UserCanManageTemplateContributorsRequest) returns (UserCanManageTemplateContributorsResponse) {
```

Comment:
Why this one, and not the one above like `UserCanManagePluginContributors` ?

Reply:
Sorry should be more clear, what's the criteria for deprecating the rpc?

Reply:
I'll deprecate that one too. I was trying to target all of the remote generation alpha RPCs.